msg/AnnotatedCameraFrame.msg

```diff
@@ -1,12 +1,6 @@
-## Constants
-# Possible values of the `rgv_id` field
-int8 RGV_1=1
-int8 RGV_2=2
-
 ## Fields
 # Time when measurement was taken, Unix time (UTC)
 time timestamp
```

Comment:
`sensor_msgs/Image` already has a time stamp under `header.stamp` so this might be unnecessary unless you want a separate timestamp to indicate when the annotation was added.